app/[locale]/hostels/page.tsx

```diff
@@ -0,0 +1,169 @@
+import { Button } from '~/components/buttons';
+import Loading from '~/components/loading';
+
+import { NotificationsList } from '../notifications';
+
+export default async function Hostels({
+  params: { locale },
+}: {
+  params: { locale: string };
+}) {
+  const text = (await getTranslations(locale)).Hostels;
+  //  const hostels = await db.query.hostels.findMany({
+  //    columns: { name: true, urlName: true, type: true },
+  //  });
+  const hostels = [
+    { name: 'H1 Abhimanu Bhawan', urlName: 'h1', type: 'boys' },
+    { name: 'Kalpana Chawala Bhawan', urlName: 'kach', type: 'girls' },
+  ];
+  const groupedHostels = groupBy(hostels, 'type') as Map<
+    'boys' | 'girls',
+    typeof hostels
+  >;
+
+  return (
+    <>
+      <ImageHeader title="Hostels" src="assets/hostels.jpg" />
+      <HostelList
+        hostelType="boys"
+        hostels={groupedHostels.get('boys')!}
+        locale={locale}
+        text={text.boysHostels}
+      />
+      <HostelList
+        hostelType="girls"
+        hostels={groupedHostels.get('girls')!}
+        locale={locale}
+        text={text.girlsHostels}
+      />
+      <section id="notification" className="container">
+        <Heading
+          text={text.notificationsTitle}
+          heading="h3"
+          href="#notification"
+          glyphDirection={'rtl'}
+        />
+        <section
+          className={cn(
+            `h-[384px] rounded-xl rounded-b-xl bg-background/[0.6] md:h-[512px]`,
+            'lg:rounded-t-xl lg:shadow-[0px_8px_0px_#e13f32_inset,_-12px_22px_60px_rgba(0,_43,_91,_0.15)] lg:drop-shadow-2xl',
+            'lg:px-6 lg:py-8 xl:px-8'
+          )}
+        >
+          <ScrollArea
+            type="always"
+            className={cn(
+              'h-full',
+              'px-3 py-3 md:px-5 md:py-5 lg:py-5 lg:pl-0 lg:pr-4 xl:pr-6'
+            )}
+          >
+            <ol className="space-y-2 sm:space-y-4 md:space-y-6">
+              <Suspense fallback={<Loading />} key={'hostel'}>
+                <NotificationsList category="hostel" locale={locale} />
+              </Suspense>
+            </ol>
+          </ScrollArea>
+        </section>
+      </section>
+      <section id="rules-and-conducts" className="container">
+        <Heading
+          text={text.misc}
+          heading="h3"
+          href="#rules-and-conducts"
+          glyphDirection={'ltr'}
+        />
+        <nav
+          className={cn(
+            'container',
+            'my-10 md:my-12 lg:my-16 xl:my-20',
+            'flex flex-col gap-5 lg:flex-row lg:justify-around'
+          )}
+        >
+          <Button
+            asChild
+            className={cn(
+              'flex flex-col',
+              'gap-2 md:gap-3 lg:gap-4 xl:gap-5',
+              'h-40 md:h-48 lg:h-60 lg:w-72 xl:w-80 2xl:w-96'
+            )}
+            variant="secondary"
+          >
+            <Link
+              href="https://nitkkr.ac.in/wp-content/uploads/2023/01/Hostel-rules.pdf"
```

Comment:
We should have a structure in place for this in AWS. Probably something like `/hostels/rules.pdf`

Reply:
@SGX21